pkg/matcher/annotation_matcher_test.go

```diff
@@ -627,6 +627,46 @@ func TestMatchPipelinerunAnnotationAndRepositories(t *testing.T) {
 				},
 			},
 		},
+		{
+			name:       "cel/match path by glob along with push event and target_branch info",
+			wantPRName: pipelineTargetNSName,
+			args: annotationTestArgs{
+				fileChanged: []string{".tekton/push.yaml"},
+				pruns: []*tektonv1.PipelineRun{
+					{
+						ObjectMeta: metav1.ObjectMeta{
+							Name: pipelineTargetNSName,
+							Annotations: map[string]string{
+								keys.OnCelExpression: "event == \"push" +
```

Comment:
this formatting is pretty weird, can you configured goland to avoid this ?

Reply:
Used `fmt.Sprintf`
```
keys.OnCelExpression: fmt.Sprintf(`event == "push" && target_branch == "%s" && ".tekton/*yaml".pathChanged()`, mainBranch)
```